### ISSUE TITLE
[Accessibility issue] No designator for mandatory fields in forms (MAS issue: 258655) 

### ISSUE THREAD

**Issue by ckbedwell** [grafana/grafana#73397]
**Steps to reproduce**:
Go to [play.grafana.org](https://play.grafana.org/). Navigate the Feature Showcase pane heading and select it. Navigate the expanded Dropdown and select the "Edit" menu item. Navigate the page "Edit Panel" page which appears. Navigate to "Settings" button and select it. Navigate the "Variables" tab item and select it. Navigate the "Variables" tab and "New" button. Navigate the "Variables edit" page. Navigate to 'Name and Query' edit fields. Verify the issue.

**Actual Result**:
There is no visual indicator what field is mandatory to be filled in.

**Expected Result**
There should be a visual indicator for required fields.

![image](https://github.com/grafana/grafana/assets/6906380/99e4887b-7271-48f5-94b3-d11719246818)

**Comment by sidgaikwad** [comment]
Hii, I would like to work on this issue but facing issue to reproduce the issue
<img width="941" alt="image" src="https://github.com/grafana/grafana/assets/48375212/4f3c28fe-cec5-4037-bad1-07b1642ded38">

**Comment by ashharrison90** [comment]
our `Field` component supports this 👍

**Comment by IamPriyabrato04** [comment]
Is this issue already resolved??
